form/persistence/persistence.hpp

```diff
@@ -16,7 +16,7 @@
 // forward declaration for form config
 namespace form::experimental::config {
   class output_item_config;
-  class tech_setting_config;
+  struct tech_setting_config;
```

Comment:
Is this adjustment necessary for this PR?

Reply:
See https://github.com/Framework-R-D/phlex/pull/94#discussion_r2500131167